.github/workflows/push.yml

```diff
@@ -58,128 +58,3 @@ jobs:
           fail_ci_if_error: false
           name: '${{env.CODECOV_UNIQUE_NAME}}'
 
-  Generate-Documentation:
-    runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/automated-docs'
-    steps:
-      - name: Checkout the Repository
-        uses: actions/checkout@v4
-        # with:
-        #   ref: develop
-
-      # - name: Pull latest changes from develop
-      #   run: git pull origin develop
-      
-      - name: Node.js Version
-        uses: actions/setup-node@v4
-        with:
-          node-version: '20'
-
-      - name: Restore node_modules from cache
-        id: cache-npm
-        uses: actions/cache@v4
-        env:
-          cache-name: cache-node-modules
-        with:
-          path: |
-            ~/.npm
-            node_modules
-          key: ${{ runner.os }}-generate-docs-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-generate-docs-${{ env.cache-name }}-
-            ${{ runner.os }}-generate-docs-
-            ${{ runner.os }}-
-
-      - name: Install dependencies
-        run: npm install
-
-      - name: Install TypeScript Globally and add GraphQL tag
-        run: yarn global add typescript
-      - run: yarn add graphql-tag
-
-      - name: Update Dependencies
-        run: yarn upgrade
-
-      - name: Generate Documentation of Markdown pages
-        run: |
-          npm install --global typedoc
-          npm install typedoc-plugin-markdown
-          npm install --save-dev @types/node
-          npx typedoc --entryPoints src/components src/screens --out talawa-admin-docs --plugin typedoc-plugin-markdown --theme markdown --entryPointStrategy expand --exclude "**/*.test.ts" --exclude "**/*.css"
-          
-      - name: Make Markdown Files MDX Compatible
-        run: python ./.github/workflows/md_mdx_format_adjuster.py --directory talawa-admin-docs
```

Comment:
Please remove the `./.github/workflows/md_mdx_format_adjuster.py`  script if it's not being used any more